examples/wrapper.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta name="viewport" content="width=device-width" />
+    <title>Media Chrome Standalone Controls Example</title>
+    <script type="module" src="../dist/index.js"></script>
+    <style>
+      /** add styles to prevent CLS (Cumulative Layout Shift) */
+      media-controller:not([audio]) {
+        aspect-ratio: 16 / 9; /* set container aspect ratio if preload=none */
+      }
+
+      #wrapper {
+        display: flex;
+        flex-direction: column;
+        width: 50vw;
+        align-items: stretch;
+      }
+
+      #wrapper > media-control-bar {
+        flex-grow: 0;
+        flex-shrink: 1;
+      }
+
+      #wrapper > .spacer {
+        background-color: var(--media-background-color, #000);
+        flex-grow: 1;
+      }
+
+      #wrapper > .fullscreen-overlay {
+        display: var(--fullscreen-overaly-display, none);
+        /* Can use fixed since we're fullscreen */
+        position: fixed;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        pointer-events: none;
+        color: #ffffff;
+      }
+
+      #wrapper:is(:fullscreen, :-webkit-full-screen, :-moz-full-screen, :-ms-fullscreen) {
+        --fullscreen-overaly-display: 'initial';
```

Comment:
```suggestion
        --fullscreen-overlay-display: 'initial';
```

Reply:
whoops that should also not be a string (I make this mistake all the time with CSS, esp. CSS vars. @luwes Wanna update your suggestion to include fixing not one, but two of my silly mistakes?

Reply:
updated